include/utilities.h

```diff
@@ -3,14 +3,15 @@
 
 #include <deal.II/base/utilities.h>
 #include <deal.II/base/smartpointer.h>
-#include <typeinfo>
-#include <cxxabi.h>
 #include <deal.II/base/exceptions.h>
 #include <deal.II/fe/fe.h>
-
 #include <deal.II/base/std_cxx11/shared_ptr.h>
 
+#include <typeinfo>
+#include <cxxabi.h>
 #include <sstream>
+#include <sys/ioctl.h>    // to know the number of cols and rows of a shell
```

Comment:
I would like the user chooses the length of each rows and that if the row length is too long ( greater than the shell length ) an assert fails.